### PR TITLE
[FC-120761][FE]:adding sla timer fields in fw-field-editor

### PR DESCRIPTION
**Pull request by Elayaraman**
## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)
 
## How Has This Been Tested?

This change is made to add new toggle in conversation properies (status)
when we passing a feature flag from the FE
so it will be dynamic according to the value we are passing

image
image

### CODE REVIEW

**Review by prasanna-vijayan — APPROVED** [review]

File: packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx [hunk 1]
```diff
@@ -1352,7 +1364,19 @@ export class FieldEditor {
...
                 </div>
               </span>
-              {toggle}
+              {dataItem?.choice_options?.resolution_timer !== undefined &&
+                renderToggle(
+                  dataItem.id,
+                  'resolution_timer',
+                  dataItem.choice_options.resolution_timer
+                )}
+              {dataItem?.choice_options?.pause_resolution_sla_timer !==
```
Comment (rihansiddhi) [inline packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx]: @Elayaraman Shouldn't we be checking if isSlaEnabled is true here?

Comment (Elayaraman) [reply packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx]: Yeah It should be 
I have made the changes

File: packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx [hunk 2]
```diff
@@ -1352,7 +1364,21 @@ export class FieldEditor {
                   ></fw-input>
                 </div>
               </span>
-              {toggle}
+              {isSlaEnabled &&
```
Comment (rihansiddhi) [inline packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx]: @Elayaraman Do we need to have isSlaEnabled check here as this is for resolution_timer?

Comment (Elayaraman) [reply packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx]: No need for it here 
mistakenly added

**Review by rihansiddhi — APPROVED** [review]

**Pull request merged**